app/src/app/api/openapi/json/route.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @swagger
+ * /api/openapi/json:
```

Comment:
I think there's a common practice of providing the endpoint at /openapi.json and/or /swagger.json. I think we can add redirects, though.

Reply:
Don't we always use JSON, though? Maybe just /api/openapi ?